code/controllers/subsystems/processing/overmap.dm

```diff
@@ -0,0 +1,5 @@
+PROCESSING_SUBSYSTEM_DEF(overmap)
+	name = "Overmap"
+	priority = SS_PRIORITY_OVERMAP
+	wait = 1
```

Comment:
This is too fast. If you want to make things smooth, find a way to do it with `animate()`.

Reply:
Smoothness relates to the stepping of position, not the visual offsets. Animate isn't going to help here. A lot of the time individual steps are less than a pixel.

Reply:
Animate also works very strangely and inconsistently with client offsets when tried previously.